woocommerce/woocommerce-admin-test-helper
# Add logger for tracks events

Adds logging to all WC tracks events.

cc @pmcpinto Is there any other information or features you would find useful here?

### Screenshots

<img width="571" alt="Screen Shot 2022-01-27 at 11 33 41 AM" src="https://user-images.githubusercontent.com/10561050/151402580-0d6de33f-4c5d-48e6-9bce-b050486bad4a.png">
<img width="695" alt="Screen Shot 2022-01-27 at 11 25 19 AM" src="https://user-images.githubusercontent.com/10561050/151402587-83d70478-6581-499d-9b51-2c7a00b817cf.png">


### Testing

1. Install and activate this plugin
2. Make sure you've opted into tracking
3. Visit the WooCommerce->Status->Logs page `wp-admin/admin.php?page=wc-status&tab=logs`
4. Select a `tracks-*` log from the dropdown
5. Visit various pages to ensure tracks are fired

Note that if you want to trigger the task tracks events in the above photo, you will need to delete the `woocommerce_task_list_tracked_completed_tasks` to retrigger them.
